### PR TITLE
WooCommerce 3.0 Compatibility

### PR DESCRIPTION
**Pull request by ChaseWiseman**

### CODE REVIEW

File: woocommerce/class-sv-wc-plugin.php [hunk 1]
```diff
@@ -377,8 +380,8 @@ protected function add_dependencies_admin_notices() {
 			$message = sprintf(
 				/* translators: Placeholders: %1$s - plugin name, %2$s - a PHP extension/comma-separated list of PHP extensions */
 				_n(
-					'%1$s requires the %2$s PHP extension to function. Contact your host or server administrator to configure and install the missing extension.',
-					'%1$s requires the following PHP extensions to function: %2$s. Contact your host or server administrator to configure and install the missing extensions.',
+					'%1$s requires the %2$s PHP extension to function. Contact your host or server administrator to install and configure the missing extension.',
```
Comment (bekarice) [inline woocommerce/class-sv-wc-plugin.php]: :rofl: for the curious: https://www.youtube.com/watch?v=NG06BR0uDKI

File: woocommerce/compatibility/class-sv-wc-order-compatibility.php [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+if ( ! class_exists( 'SV_WC_Order_Compatibility' ) ) :
+
+/**
+ * WooCommerce order compatibility class.
+ *
+ * @since 4.6.0-dev
+ */
+class SV_WC_Order_Compatibility extends SV_WC_Data_Compatibility {
+
+
```
Comment (maxrice) [inline woocommerce/compatibility/class-sv-wc-order-compatibility.php]: should indicate here what order this array is in, i'm assuming `new property => old property`?

File: woocommerce/compatibility/class-sv-wc-order-compatibility.php [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+			if ( ! $order->get_id() ) {
+				$order->save();
+			}
+
+			$item->set_order_id( $order->get_id() );
+			$item->set_props( $args );
+			$item->save();
+
+			do_action( 'woocommerce_order_update_fee', $order->get_id(), $item->get_id(), $args );
```
Comment (maxrice) [inline woocommerce/compatibility/class-sv-wc-order-compatibility.php]: are these actions not triggered in WC 2.7?

File: woocommerce/compatibility/class-sv-wc-product-compatibility.php [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+if ( ! class_exists( 'SV_WC_Product_Compatibility' ) ) :
+
+/**
+ * WooCommerce product compatibility class.
+ *
+ * @since 4.6.0-dev
+ */
+class SV_WC_Product_Compatibility extends SV_WC_Data_Compatibility {
+
+
```
Comment (maxrice) [inline woocommerce/compatibility/class-sv-wc-product-compatibility.php]: same comment as above, clarify what order this array is in

**Pull request merged**